api/v1/app.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python3
+"""DOCSTRING"""
+from flask import Flask, jsonify
+from models import storage
+from api.v1.views import app_views
+
+
+app = Flask(__name__)
+app.register_blueprint(app_views, url_prefix='/api/v1')
+
+
+@app.teardown_appcontext
+def teardown_fs_db(exception):
+    """closes the storage on teardown"""
+    storage.close()
+
+@app.errorhandler(404)
+def api_error(self):
+    """Returns an error page when pinged"""
+    return jsonify({ "error": "Not Found" }), 404
```

Comment:
minor nitpick, but is the spaces on the inside of the curly braces needed? It looks odd to me